Adafruit_AW9523.cpp

```diff
@@ -292,3 +292,27 @@ bool Adafruit_AW9523::openDrainPort0(bool od) {
 
   return opendrain.write(!od);
 }
+
+/*!
+ * From the data sheet: 256 step dimming range select
+ * 00：0~IMAX
+ * 01：0~(IMAX×3/4)
+ * 10：0~(IMAX×2/4)
+ * 11：0~(IMAX×1/4)
```

Comment:
How about adding a typedef enum for these with descriptive names? That way it'll be a little more meaningful in terms of actual current than just the raw register value. So instead of this:
```cpp
aw.configureLEDCurrent(2)
```
could do this:
```cpp
aw.configureLEDCurrent(AW9523_IMAX_18MA);
```
From a quick scan of the datasheet, I think the base 37mA is set?

Reply:
For this specific chip it is indeed set to 37 mA for some unspecified, non-obvious reason.  I hate to hardcode values into enums/constants, but if there's a newer chip with the same gross granularity but a different Imax, I think the code ought to have to change rather than silently providing too little or too much current.  After all, the whole reason I'm writing this was too much current for the cheap LEDs that came with my $20 Arduino/Pi starter kit.